parl/algorithms/fluid/maddpg.py

```diff
@@ -22,13 +22,13 @@
 
 __all__ = ['MADDPG']
 
-from gym import spaces
 from parl.core.fluid.policy_distribution import SoftCategoricalDistribution
 from parl.core.fluid.policy_distribution import SoftMultiCategoricalDistribution
 
 
 def SoftPDistribution(logits, act_space):
```

Comment:
Could you add some comments about this function?

Reply:
done